src/com/palmergames/bukkit/towny/object/TownyWorld.java

```diff
@@ -645,21 +647,22 @@ public int getMinDistanceFromOtherTowns(Coord key) {
 	public int getMinDistanceFromOtherTowns(Coord key, Town homeTown) {
 
 		double min = Integer.MAX_VALUE;
-		for (Town town : getTowns())
+		for (Map.Entry<String, Town> mapElement : getTowns().entrySet()) {

```

Comment:
This should be changed to `for (Town town : getTowns.values())` as the key in the `Map.Entry` is never used.